File: ApiSuvesaPos/Negocio/Modelo/configuraciones/mensajereceptor.vb
Namespace Modelo
    Public Class mensajereceptor
        Public Property Id As Integer
        Public Property Clave As String
        Public Property NumeroCedulaEmisor As String
        Public Property FechaEmisionDoc As Date
        Public Property Mensaje As String
        Public Property MontoTotalImpuesto As Decimal
        Public Property TotalFactura As Decimal
        Public Property Estado As String
        Public Property Consecutivo As String
        Public Property Proveedor As String
        Public Property Moneda As String
        Public Property FechaComprobante As Date
        Public Property PDF As String
        Public Property DetalleMensaje As String
        Public Property CodigoActividad As String
        Public Property CondicionImpuesto As String
        Public Property MontoTotalImpuestoAcreditar As Decimal
        Public Property MontoTotalDeGastoAplicable As Decimal
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/configuraciones/mensajereceptor.cs
using System;

namespace Modelo
{
    public class mensajereceptor
    {
        public int Id { get; set; }
        public string Clave { get; set; }
        public string NumeroCedulaEmisor { get; set; }
        public DateTime FechaEmisionDoc { get; set; }
        public string Mensaje { get; set; }
        public decimal MontoTotalImpuesto { get; set; }
        public decimal TotalFactura { get; set; }
        public string Estado { get; set; }
        public string Consecutivo { get; set; }
        public string Proveedor { get; set; }
        public string Moneda { get; set; }
        public DateTime FechaComprobante { get; set; }
        public string PDF { get; set; }
        public string DetalleMensaje { get; set; }
        public string CodigoActividad { get; set; }
        public string CondicionImpuesto { get; set; }
        public decimal MontoTotalImpuestoAcreditar { get; set; }
        public decimal MontoTotalDeGastoAplicable { get; set; }
    }
}
